doc-content/enterprise-only/admin-and-config/managing-business-central-project-preferences-proc.adoc

```diff
@@ -0,0 +1,80 @@
+[id='managing-business-central-project-preferences-proc']
+= Customizing project preferences
+
+A project stores assets and is part of a space. A space can hold multiple projects.
+
+For example, an organization has many departments, such as HR, Payroll, Engineering, R&D, and so on. All these departments map to spaces. Every department can have one or more processes that map to projects running at the same time.
```

Comment:
Instead of  `All these departments map to spaces.' I'd prefer: `Each department maps to a space. And every department could have their own projects".

Projects running at the same time is a bit confuse.. it sounds like runtime environment